support/cas-server-support-oidc/src/test/java/org/apereo/cas/oidc/web/OidcImplicitIdTokenAndTokenAuthorizationResponseBuilderTests.java

```diff
@@ -34,6 +35,7 @@
  * @since 6.2.0
  */
 @Tag("OIDC")
+@TestPropertySource(properties = "cas.authn.oauth.accessToken.createAsJwt=true")
```

Comment:
I believe OIDC expects JWT ticket, right? Also after this PR createAsJwt have to be set to true to pass the unit test. (Originally OidcIdTokenGeneratorService would not be able to get the encoded access token, but after further polishing up this requirement is lifted.)